client/src/components/Survey.test.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Survey, SurveyProps } from './Survey';
+import { PostCallConfig } from '../models/ConfigModel';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+const mockPostCall: PostCallConfig = {
+  survey: {
+    type: 'msforms',
+    options: {
+      surveyUrl: 'dummySurveyUrl'
+    }
+  }
+};
+
+describe('Survey', () => {
+  it('should render', async () => {
+    const survey = await mount<SurveyProps>(<Survey postCall={mockPostCall} onRejoinCall={jest.fn()} />);
+    expect(survey.find('iframe').length).toBe(1);
+    expect(survey.props().postCall?.survey?.type).toBe(mockPostCall.survey?.type);
+    expect(survey.props().postCall?.survey?.options.surveyUrl).toBe(mockPostCall.survey?.options.surveyUrl);
```

Comment:
Not sure how useful it is: we are setting the prop and then checking that the prop was set correctly. Are we testing any code that we wrote as part of this? It doesn't seem so.

A more useful check would be to verify that iframe's url is set to the value we expect.

Reply:
Have made this change, the iframe's url will now be tested to match the one passed in on props of Survey